.drone.yml

```diff
@@ -21,6 +21,20 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: package-images
+  image: rancher/dapper:v0.5.5
+  commands:
+    - docker pull --quiet rancher/hardened-build-base:v1.16.4b7
```

Comment:
The release-1.20 branch should be on hardened-build-base:v1.15.x (same version as upstream is using to build Kubernetes)

Reply:
that's news to me... thought it always needed to be latest.. no problem tho